selfdrive/modeld/SConscript

```diff
@@ -87,10 +87,12 @@ if use_thneed and arch == "larch64" or GetOption('pc_thneed'):
     "#tinygrad_repo/extra/utils.py",
     "#tinygrad_repo/tinygrad/llops/ops_gpu.py",
     "#tinygrad_repo/tinygrad/llops/ops_opencl.py",
+    "#tinygrad_repo/tinygrad/graph.py",
```

Comment:
can replace this list with scons's `env.Glob` or read this list from `release/files_common`

Reply:
Ah yea I should probably just read it from release